fvtest/compilertriltest/LogicalTest.cpp

```diff
@@ -211,5 +215,69 @@ INSTANTIATE_TEST_CASE_P(LogicalTest, Int64LogicalBinary, ::testing::Combine(
 		std::make_tuple("land", land),
 		std::make_tuple("lxor", lxor)
 				)));
+<<<<<<< HEAD
```

Comment:
Seems like some leftovers from a merge conflict got missed :slightly_smiling_face:. Please remove.

Reply:
I think just removing the last commit from this PR should fix this.

Reply:
Removed the last commit and checked that there is no conflict in merging. Hope now this works fine.